core/pkg/eval/json_evaluator.go

```diff
@@ -59,6 +59,8 @@ func NewJSONEvaluator(logger *logger.Logger, s *store.Flags) *JSONEvaluator {
 		jsonEvalTracer: otel.Tracer("jsonEvaluator"),
 	}
 	jsonlogic.AddOperator("fractionalEvaluation", ev.fractionalEvaluation)
+	jsonlogic.AddOperator("starts_with", ev.startsWithEvaluation)
```

Comment:
I just double-checked JSON logic, and it looks like they use snake case. As a follow-up, it may be worth adding an alias for the `fractionEvaluation` operator to also follow this convention.